app/src/views/OperationalLearning/OperationalLearningMap/index.tsx

```diff
@@ -0,0 +1,254 @@
+import {
+    useCallback,
+    useMemo,
+    useState,
+} from 'react';
+import {
+    Container,
+    NumberOutput,
+    TextOutput,
+} from '@ifrc-go/ui';
+import { useTranslation } from '@ifrc-go/ui/hooks';
+import { maxSafe } from '@ifrc-go/ui/utils';
+import {
+    _cs,
+    isDefined,
+    isNotDefined,
+} from '@togglecorp/fujs';
+import {
+    MapLayer,
+    MapSource,
+} from '@togglecorp/re-map';
+
+import BaseMap from '#components/domain/BaseMap';
+import Link from '#components/Link';
+import MapContainerWithDisclaimer from '#components/MapContainerWithDisclaimer';
+import MapPopup from '#components/MapPopup';
+import useCountry from '#hooks/domain/useCountry';
+import {
+    COLOR_BLUE,
+    COLOR_LIGHT_BLUE,
+} from '#utils/constants';
+import {
+    adminFillLayerOptions,
+    getPointCircleHaloPaint,
+} from '#utils/map';
+import { GoApiResponse } from '#utils/restRequest';
+
+import i18n from './i18n.json';
+import styles from './styles.module.css';
+
+type learningStatsResponse = GoApiResponse<'/api/v2/ops-learning/stats/'>;
+
+const sourceOptions: mapboxgl.GeoJSONSourceRaw = {
+    type: 'geojson',
+};
+
+interface CountryProperties {
+    country_id: number;
+    name: string;
+    operation_count: number;
+}
+interface ClickedPoint {
+    feature: GeoJSON.Feature<GeoJSON.Point, CountryProperties>;
+    lngLat: mapboxgl.LngLatLike;
+}
+interface Props {
+   className?: string;
+   learning: learningStatsResponse | undefined;
+}
+const LEARNINGS_LOW_COLOR = COLOR_LIGHT_BLUE;
+const LEARNINGS_HIGH_COLOR = COLOR_BLUE;
+
+function getMaxLearning(learning: learningStatsResponse) {
+    const learningData = learning?.learning_by_country.filter(
+        ({ count }) => isDefined(count),
+    );
+
+    const maxLearning = maxSafe(
+        learningData?.map(({ count }) => count),
+    );
+    return maxLearning;
+}
```

Comment:
Move this function inside to directly calculate getMaxLearning

Reply:
Agree. We do not need a separate function that cannot be reused anywhere else.